executor/batch_point_get.go

```diff
@@ -161,6 +170,7 @@ func (e *BatchPointGetExec) Next(ctx context.Context, req *chunk.Chunk) error {
 	}
 
 	err := FillVirtualColumnValue(e.virtualColumnRetFieldTypes, e.virtualColumnIndex, e.schema, e.columns, e.ctx, req)
+	e.memTracker.Consume(req.MemoryUsage())
```

Comment:
The chunk's memory should be mamaged by the parent executor of BatchPointGetExec..  So please remove this line...

Reply:
mysql> explain select * from t1 where a in (1,2,3);
+-------------------+---------+------+---------------+----------------------------------------------+
| id                | estRows | task | access object | operator info                                |
+-------------------+---------+------+---------------+----------------------------------------------+
| Batch_Point_Get_1 | 3.00    | root | table:t1      | handle:[1 2 3], keep order:false, desc:false |
+-------------------+---------+------+---------------+----------------------------------------------+
1 row in set (4.62 sec)
in this case,the BatchPointGetExec does not have parent executor，i think need add bool to mark if have parent executor

Reply:
I don't think so. If it doesn't have parent executor, the chunk will be sent to client....  So the memory shouldn't be track..